core/workspace.js

```diff
@@ -13,62 +13,69 @@
 goog.module('Blockly.Workspace');
 goog.module.declareLegacyNamespace();
 
+/* eslint-disable-next-line no-unused-vars */
+const Abstract = goog.requireType('Blockly.Events.Abstract');
```

Comment:
This may be clearer as EventsAbstract, but up to you.

Reply:
I like that change, but we're not doing that in other files (and I prefer to be consistent).
I'll create a bug for updating the require in all the files at once and leave this for now.

Reply:
https://github.com/google/blockly/issues/5336